lib/package/commands.coffee

```diff
@@ -111,6 +111,12 @@ module.exports =
       'julia-debug:continue': (ev) => juno.runtime.debugger.continueForward(ev)
       'julia-debug:open-debugger-pane': => juno.runtime.debugger.open()
 
+      'julia:new-julia-file': =>
+        atom.workspace.open().then((ed) =>
+          gr = atom.grammars.grammarForScopeName('source.julia')
+          return ed.destroy() unless gr
```

Comment:
Why `destroy` the editor here?

Reply:
Because then it's not "julia file" and so we want to show that this command failed ?
Notifying that `atom-language-julia` isn't installed in this path would be more friendly, though.

Reply:
Yeah, I feel like keeping the editor open and showing a warning or whatever is a bit nicer.

Reply:
please change the notification wording as you want :)

Reply:
Actually, we're checking whether language-julia is installed on start [here](https://github.com/JunoLab/atom-julia-client/blob/master/lib/julia-client.coffee#L50-L59), I guess we can remove all checks here.

Reply:
yeah, I borrowed the notification message from there, and felt like we are doing something boilerplate.

Let's just go with `ed.setGrammar(gr) if gr` then.